### PR TITLE
Doc type removal

### PR DESCRIPTION
**Pull request by pwnbus**
We're positioning ourselves to upgrade to ES6, which no longer uses multiple _type values.

### CODE REVIEW

File: tests/mq/plugins/test_suricataFixup.py [hunk 1]
```diff
@@ -115,7 +115,8 @@ def test_suricata_notype_log(self):
         assert result['category'] == 'suricata'
```
Comment (Phrozyn) [inline tests/mq/plugins/test_suricataFixup.py]: This file has a lot of assertions for doc_type that are not addressed. Some are doc_type = 'event' some are doc_type = 'nsm'  and they are not on the tests that check for doc_type being incorrect.

File: meteor/client/alertdetails.html [hunk 2]
```diff
@@ -58,7 +58,6 @@
         <tr>
             <td>{{documentindex}}</td>
             <td>{{documentid}}</td>
-            <td>{{documenttype}}</td>
```
[the remark this answers is not in the document]
Comment (pwnbus) [reply meteor/client/alertdetails.html]: We removed this field all together from the alert/event that gets saved to mongo (as part of https://github.com/mozilla/MozDef/pull/1202/files#diff-e5d244702192ca178405533ecb3868f5L412) , so IMO this field isn't necessary anymore to show in the web UI.

File: cron/okta2mozdef.py [hunk 3]
```diff
@@ -118,7 +105,7 @@ def main():
                             mozdefEvent['details'].pop('action')
 
                             jbody=json.dumps(mozdefEvent)
```
[the remark this answers is not in the document]
Comment (pwnbus) [reply cron/okta2mozdef.py]: Fixed in https://github.com/mozilla/MozDef/pull/1202/commits/02cc88d2eb1397fc3f6080313519f903ab3a01bb

File: tests/mq/plugins/test_suricataFixup.py (hunk 1 above)
Comment (pwnbus) [reply tests/mq/plugins/test_suricataFixup.py]: Based off our conversation off line, I think we're good here (and these will get officially cleaned up when we remove doc_type entirely due to ES6)

**Pull request merged**